From 965fe6c5442b95b61f616caefa68cf2109d12ccc Mon Sep 17 00:00:00 2001
From: Michael Scott <mike@foundries.io>
Date: Mon, 8 May 2023 17:45:30 -0700
Subject: [PATCH 5/6] imx8mn_var_som: adjust SPL stack settings to match
 imx8mn_evk

mirrors commit 8eac078fc59571dacb0f2d9623bf8384288a157b
for imx8mn_evk

Signed-off-by: Michael Scott <mike@foundries.io>
---
 include/configs/imx8mn_var_som.h | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/include/configs/imx8mn_var_som.h b/include/configs/imx8mn_var_som.h
index b260ce9c8b5..6f55453a642 100644
--- a/include/configs/imx8mn_var_som.h
+++ b/include/configs/imx8mn_var_som.h
@@ -34,8 +34,9 @@
 #define CONFIG_SYS_UBOOT_BASE		(QSPI0_AMBA_BASE + CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR * 512)
 
 #ifdef CONFIG_SPL_BUILD
-#define CONFIG_SPL_STACK		0x95dff0
-#define CONFIG_SPL_BSS_START_ADDR	0x95e000
+#define CONFIG_SPL_STACK		0x96fff0
+#define CONFIG_SPL_BSS_START_ADDR	0x954000
+#define CONFIG_MALLOC_F_ADDR 0x970000
 #define CONFIG_SPL_BSS_MAX_SIZE	SZ_8K		/* 8 KB */
 #define CONFIG_SYS_SPL_MALLOC_START	0x42200000
 #define CONFIG_SYS_SPL_MALLOC_SIZE	SZ_16M
-- 
2.40.1

